build/common/installer/scripts/tomlparser-common-agent-config.rb

```diff
@@ -62,12 +63,25 @@ def populateSettingValuesFromConfigMap(parsedConfig)
   end
 end
 
+def getKubernetesMetadataCacheTTL(parsedConfig)
+  begin
+    if !parsedConfig.nil? && !parsedConfig[:agent_settings].nil?
+      k8s_metadata_config = parsedConfig[:agent_settings][:k8s_metadata_config]
+      if !k8s_metadata_config.nil? && !k8s_metadata_config[:kube_meta_cache_ttl].nil?
```

Comment:
since the units of this field is seconds, hence have the seconds prefix of this config entry, kube_meta_cache_ttl_secs

Reply:
changed